src/main/java/com/chatapp/vartalaab/dto/MessageDto.java

```diff
@@ -1,14 +1,12 @@
 package com.chatapp.vartalaab.dto;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
 import lombok.Data;
 
 import java.util.Date;
 
 @Data
 public class MessageDto {
 
-    @JsonIgnore
```

Comment:
why this has been removed is there an schema change ?

Reply:
Not exactly schema change, but kinda made mandatory to send sender username from UI, otherwise we had to write new dto which had almost similar attributes.